components/custom-network/token-configuration.tsx

```diff
@@ -198,38 +183,55 @@ export default function TokenConfiguration({
 
         <div className="col-3 pt-2">
           <Button onClick={handleShowModal}>
-            {t("custom-network:steps.token-configuration.actions.deploy-nft-token")}
+            {t("custom-network:steps.token-configuration.actions.deploy-new-token")}
           </Button>
         </div>
       </div>
 
-      <div className="row align-items-center">
-        <div className="form-group col-12">
+      <div className="row">
+        <div className="form-group col-6">
           <label className="caption-small mb-2">
-            {t("custom-network:steps.token-configuration.fields.nft-token-uri.label")}
+            {t("custom-network:steps.token-configuration.fields.name.label")}
           </label>
+          <input type="text" className="form-control" value={networkToken?.name}  readOnly />
+        </div>
 
-          <input 
-            type="text" 
-            className="form-control" 
-            value={bountyTokenUri}
-            onChange={handleNFTTokenURIChange}
-            onBlur={validateTokenUri}
-          />
-
-          {
-            bountyTokenUriError && 
-            <small className="small-info text-danger">
-              {t("custom-network:steps.token-configuration.fields.nft-token-uri.error")}
-            </small>
-          }
+        <div className="form-group col-6">
+          <label className="caption-small mb-2">
+          {t("custom-network:steps.token-configuration.fields.symbol.label")}
+          </label>
+          <input type="text" className="form-control" value={networkToken?.symbol}  readOnly />
         </div>
       </div>
 
-      <DeployNFTModal 
+      <div className="row">
+          <>
+            <MultipleTokensDropdown
+              label={t("select-multiple.allowed-transactional-tokens")}
+              description={t("select-multiple.add-transactional-tokens")}
+              addToken={addTransactionalToken}
+              tokens={handleEmptyTokens(allowedTransactionalTokens)}
+              canAddToken={false}
```

Comment:
shouldn't this depend on the user wallet instead of hardcoded to false ?

Reply:
not in this situation because it is creating the network and can only use allowed tokens